src/components/BaseAlert.vue

```diff
@@ -35,10 +38,37 @@ export default {
         return ['success', 'error', 'warning', 'info'].includes(value);
       },
     },
+    /**
+     * Enable Inline version.
+     */
+    inline: Boolean,
+    /**
+     * Disable close button.
+     */
+    disableClose: Boolean,
+    /**
+     * Show the alert icon.
+     */
+    showIcon: Boolean,
   },
   computed: {
     classObject() {
-      return `alert--${this.variant}`;
+      return `alert--${this.variant} ${this.inline ? 'alert--inline' : ''} ${this.icon ? 'alert--icon' : ''}`;
```

Comment:
This is fine - but for future reference you can write this as:

```suggestion
      return [`alert--${this.variant}`, {
        'alert--inline': this.inline,
        'alert--icon': this.icon,
      }];
```

I think it's a tiny bit more readable.

Reply:
Updated